README.md

```diff
@@ -8,7 +8,7 @@
 ## Built With
 
 - HTML & CSS
-- Github
+- Git & Github
 
 ## Live Demo (if available)
 
```

Comment:
- [ ]  I can see that live demo section in readme is empty, if so I suggest you to delete this section, in the future when you start deploying projects you can insert link here 👍🏻